### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bwilk**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
You can also use elements from these templates:

Bug.md
```
### Summary

(Summarize the bug encountered concisely)

### Steps to reproduce

(How one can reproduce the issue - this is very important)

### Expected behavior

(What you should see instead)

### Actual behavior

(What actually happens)

### Relevant logs and/or screenshots

(Paste any relevant logs - please use code blocks (```) to format console output,
logs, and code as it's very hard to read otherwise.)
```
Feature proposal:
```
### Description

(Include problem, use cases, benefits, and/or goals)

### Proposal

### Links / references
```

Merge request:

```
### Summary

(Provide a general description of the code changes in your pull
request... were there any bugs you had fixed? If so, mention them. If
these bugs have open issues, be sure to tag them here as well, to keep the
conversation linked together.)

### Other Information

(If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.)

(Please add the CHANGELOG entry into appropriate subsection in "Unreleased"
section.)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
[the remark this answers is not in the document]
Comment (bwilk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Doesn't it overlap with **Describe the bug** section ??

[the remark this answers is not in the document]
Comment (mkasztelnik) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Not really. The intent of this section is to describe what is wrong, no how to reproduce the error.

[the remark this answers is not in the document]
Comment (bwilk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: To describe how to reproduce you simply use **To Reproduce** section. **Describe the bug** is the place to describe actual behaviour

**Pull request merged**